src/schema/v2/checkout.ts

```diff
@@ -0,0 +1,413 @@
+import {
+  GraphQLFieldConfig,
+  GraphQLString,
+  GraphQLObjectType,
+  graphql,
+  GraphQLBoolean,
+  GraphQLList,
+} from "graphql"
+import gql from "lib/gql"
+import { ResolverContext } from "types/graphql"
+import { schema } from "schema/v2"
+
+const CheckoutStageType: GraphQLObjectType = new GraphQLObjectType({
+  name: "CheckoutStage",
+  fields: {
+    stage: {
+      type: GraphQLString,
+      description: "The stage of the checkout",
+      resolve: (stage) => {
+        return stage.stage
+      },
+    },
+    knownRequired: {
+      type: GraphQLBoolean,
+      description:
+        "Indicates if the stage is known to be required based on current order data",
+      resolve: (stage) => {
+        return stage.knownRequired
+      },
+    },
+    savedValues: {
+      type: GraphQLString,
+      description: "The currently saved values for the stage",
+      resolve: (stage) => {
+        return JSON.stringify(stage.savedValues)
+      },
+    },
+    options: {
+      type: GraphQLString,
+      description: "The options for the stage",
+      resolve: (stage) => {
+        return JSON.stringify(stage.options)
+      },
+    },
```

Comment:
These values could have better types based on what stage it is, and checkout form fields could match them.